addons/transifex/models/transifex_code_translation.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import psycopg2
+import werkzeug.urls
+from configparser import ConfigParser
+from os import pardir
+from os.path import isfile, join as opj
+
+import odoo
+from odoo import models, fields, _
+from odoo.tools.translate import CodeTranslations
+
+
+class TransifexCodeTranslation(models.Model):
+    _name = "transifex.code.translation"
+    _description = "Code Translation"
+    _log_access = False
+
+    source = fields.Text(string='Code')
+    value = fields.Text(string='Translation Value')
+    module = fields.Char(help="Module this term belongs to")
+    lang = fields.Selection(selection='_get_languages', string='Language', validate=False)
+    transifex_url = fields.Char("Transifex URL", compute='_compute_transifex_url', store=True, precompute=True,
+                                help="Propose a modification in the official version of Odoo")
+
+    def _auto_init(self):
+        super()._auto_init()
+        self.env.cr.execute(f'DELETE FROM {self._table}')
+
+    def _get_languages(self):
+        return self.env['res.lang'].get_installed()
+
+    def _load_code_translations(self, module_names=None, langs=None):
+        try:
+            # the table lock promises translations for a (module, language) will only be created once
+            self.env.cr.execute(f'LOCK TABLE {self._table} IN EXCLUSIVE MODE NOWAIT')
+
+            if module_names is None:
+                module_names = self.env['ir.module.module'].search([('state', '=', 'installed')]).mapped('name')
+            if langs is None:
+                langs = [lang for lang, _ in self.env['res.lang'].get_installed() if lang != 'en_US']
```

Comment:
You could reuse `_get_languages` above
```suggestion
                langs = [lang for lang, _ in self._get_languages() if lang != 'en_US']
```